src/dao/services/contracts/MultiSigDAOService.ts

```diff
@@ -170,15 +172,18 @@ interface UpdateDAODetailsTransactionParams {
   name: string;
   description: string;
   logoUrl: string;
+  infoUrl: string;
+  type: string;
   webLinks: string[];
   daoAccountId: string;
   signer: HashConnectSigner;
 }
 
 async function sendUpdateDAODetailsTransaction(params: UpdateDAODetailsTransactionParams) {
-  const { name, description, logoUrl, webLinks, daoAccountId, signer } = params;
+  const { name, description, logoUrl, infoUrl, type, webLinks, daoAccountId, signer } = params;
   const contractInterface = new ethers.utils.Interface(BaseDAOJSON.abi);
-  const updateDaoParams: any[] = [name, logoUrl, description, webLinks];
+  const daoInfoUrl = type === DAOType.MultiSig ? "" : infoUrl;
+  const updateDaoParams: any[] = [name, logoUrl, daoInfoUrl, description, webLinks];
   const data = contractInterface.encodeFunctionData(BaseDAOContractFunctions.UpdateDAOInfo, updateDaoParams);
```

Comment:
we can also avoid having condition in service file by checking `DAOType` in `UpdateDAODetails` form in the mutate call

`async function onSubmit(data: SettingsForm) {
    mutate({
      name: data.name,
      description: data.description,
      logoUrl: data.logoUrl,
      infoUrl: data.type === DAOType.MultiSig ? "" : data.infoUrl,
      type: data.type,
      webLinks: isEmpty(data.webLinks) ? [] : data.webLinks.map((link) => link.value),
      daoAccountId,
    });
  }`

Reply:
Te above call is not required now, since we have info URL for multisig as well.

Reply:
Yes. Removed it.